styles/readwise/catppuccin.user.css

```diff
@@ -0,0 +1,186 @@
+/* ==UserStyle==
+@name Readwise Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/readwise
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/readwise
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/readwise/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Areadwise
+@description Soothing pastel theme for Readwise
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document url-prefix('https://readwise.io/')
+{
+  @media (prefers-color-scheme: light) {
+    :root {
+      #catppuccin(@lightFlavor, @accentColor);
+    }
+  }
+  @media (prefers-color-scheme: dark) {
+    :root {
+      #catppuccin(@darkFlavor, @accentColor);
+    }
+  }
+
+  :root.theme--dark {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root.theme--light {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+
+  @media (prefers-color-scheme: dark) {
+    :root:not(.theme--light, .theme--dark) .readwise-header-logo {
+      // Invert logo color for those pages
+      filter: invert(1);
+    }
+  }
```

Comment:
Can we theme the logo with https://github.com/catppuccin/userstyles/blob/main/docs/tips-and-tricks.md#how-do-i-theme-images-and-svgs?

Reply:
The change is introduced, but I have a weird issue: escape function throws a weird error when `<` or `>` is used. So as a hacky workaround I've manually encoded this SVG. But a help to fix that is appreciated.
<img width="902" alt="Screenshot 2024-08-02 at 14 32 30" src="https://github.com/user-attachments/assets/c7e28c69-cb89-43aa-ad39-6374595db765">